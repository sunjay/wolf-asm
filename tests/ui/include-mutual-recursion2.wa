.include 'include-mutual-recursion3.wa'
