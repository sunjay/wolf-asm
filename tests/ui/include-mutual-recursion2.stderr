error: maximum `.include` recursion depth reached while reading files:
    51. ../tests/ui/include-mutual-recursion1.wa
    50. ../tests/ui/include-mutual-recursion3.wa
    49. ../tests/ui/include-mutual-recursion2.wa
    48. ../tests/ui/include-mutual-recursion1.wa
    47. ../tests/ui/include-mutual-recursion3.wa
    46. ../tests/ui/include-mutual-recursion2.wa
    45. ../tests/ui/include-mutual-recursion1.wa
    44. ../tests/ui/include-mutual-recursion3.wa
    43. ../tests/ui/include-mutual-recursion2.wa
    42. ../tests/ui/include-mutual-recursion1.wa
    41. ../tests/ui/include-mutual-recursion3.wa
    40. ../tests/ui/include-mutual-recursion2.wa
    39. ../tests/ui/include-mutual-recursion1.wa
    38. ../tests/ui/include-mutual-recursion3.wa
    37. ../tests/ui/include-mutual-recursion2.wa
    36. ../tests/ui/include-mutual-recursion1.wa
    35. ../tests/ui/include-mutual-recursion3.wa
    34. ../tests/ui/include-mutual-recursion2.wa
    33. ../tests/ui/include-mutual-recursion1.wa
    32. ../tests/ui/include-mutual-recursion3.wa
    31. ../tests/ui/include-mutual-recursion2.wa
    30. ../tests/ui/include-mutual-recursion1.wa
    29. ../tests/ui/include-mutual-recursion3.wa
    28. ../tests/ui/include-mutual-recursion2.wa
    27. ../tests/ui/include-mutual-recursion1.wa
    26. ../tests/ui/include-mutual-recursion3.wa
    25. ../tests/ui/include-mutual-recursion2.wa
    24. ../tests/ui/include-mutual-recursion1.wa
    23. ../tests/ui/include-mutual-recursion3.wa
    22. ../tests/ui/include-mutual-recursion2.wa
    21. ../tests/ui/include-mutual-recursion1.wa
    20. ../tests/ui/include-mutual-recursion3.wa
    19. ../tests/ui/include-mutual-recursion2.wa
    18. ../tests/ui/include-mutual-recursion1.wa
    17. ../tests/ui/include-mutual-recursion3.wa
    16. ../tests/ui/include-mutual-recursion2.wa
    15. ../tests/ui/include-mutual-recursion1.wa
    14. ../tests/ui/include-mutual-recursion3.wa
    13. ../tests/ui/include-mutual-recursion2.wa
    12. ../tests/ui/include-mutual-recursion1.wa
    11. ../tests/ui/include-mutual-recursion3.wa
    10. ../tests/ui/include-mutual-recursion2.wa
     9. ../tests/ui/include-mutual-recursion1.wa
     8. ../tests/ui/include-mutual-recursion3.wa
     7. ../tests/ui/include-mutual-recursion2.wa
     6. ../tests/ui/include-mutual-recursion1.wa
     5. ../tests/ui/include-mutual-recursion3.wa
     4. ../tests/ui/include-mutual-recursion2.wa
     3. ../tests/ui/include-mutual-recursion1.wa
     2. ../tests/ui/include-mutual-recursion3.wa
     1. ../tests/ui/include-mutual-recursion2.wa

error: aborting due to 1 previous error

